help/destinations/catalog/crm/outreach.md

```diff
@@ -0,0 +1,208 @@
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the Outreach destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the Outreach destination fields, follow these steps:
+
+1. In the Mapping step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Add new mapping](../../assets/catalog/crm/outreach/add-new-mapping.png)
+
+1. In the select source field window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+![Source mapping](../../assets/catalog/crm/outreach/source-mapping.png)
+
+1. In the select target field window, select the target field and choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+![Target mapping using contactId](../../assets/catalog/crm/outreach/target-mapping.png)
+
+1. For custom attributes, in the select target field window, select the target field and choose the **[!UICONTROL Select custom attributes]** category, Next provide the desired target attribute name and add the mappings desired.
+![Target mapping using LastName](../../assets/catalog/crm/outreach/target-mapping-lastname.png)
+
+1. For instance, you could add the following mapping between your XDM profile schema and your [!DNL Outreach] instance:
+
+    ||XDM Profile Schema|[!DNL Outreach] Instance| Mandatory|
+    |---|---|---|---|
+    |Attributes|<ul><li><code>person.name.firstName</code></li><li><code>person.name.lastName</code></li></ul>|<ul><li><code>firstName</code></li><li><code>lastName</code></li></ul>|
+    |Identities|<ul><li><code>Oid</code></li></ul>|<ul><li><code>OutreachId</code></li></ul>|Yes|
+
+1. An example using these mappings is shown below:
+![Target mapping](../../assets/catalog/crm/outreach/mappings.png)
+
+Refer to [Outreach prospect documentation](https://api.outreach.io/api/v2/docs#prospect) for a comprehensive list of supported attributes.
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.html?lang=en#scheduling) step you must manually map Platform segments to the custom field attribute in Outreach.
+
+To do this, select each segment, then enter the corresponding numeric value which corresponds to the *custom`N`* attribute from Outreach in the **[!UICONTROL Mapping ID]** field.
```

Comment:
Let's add a separate section explaining how to obtain this mapping ID, step by step, with screenshots from the Outreach UI.

Reply:
Have updated the content, the rewording should hopefully fix this explanation discrepancy. 
P.s. We already have screengrab's in pre-requisites explaining creation of custom fields.